x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/utils/enrichments/index.ts

```diff
@@ -34,9 +34,20 @@ export const enrichEvents: EnrichEventsFunction = async ({
     logger.debug('Alert enrichments started');
     const isNewRiskScoreModuleAvailable = experimentalFeatures?.riskScoringRoutesEnabled ?? false;
 
+    // TODO: Remove isNewRiskScoreModuleInstalled once the legacy risk score module is removed
```

Comment:
I think that this comment is a little too vague to be a TODO. Should the entire definition of `isNewRiskScoreModuleInstalled` be removed, including the new call to `getIsHostRiskScoreAvailable`? If so, what replaces it?